content/chainguard/chainguard-images/about/_index.md

```diff
@@ -38,9 +38,9 @@ tutorials: [
     url: "/chainguard/chainguard-images/working-with-images/about/images-compiled-programs/compiled-programs"
   },
   {
-    title: "Container Variants",
+    title: "Development Containers",
```

Comment:
Do we want this to say "Container Variants" again since the link title for the article is "Container Variants"?

Reply:
Yes, forgot to update the URL here too